replica/database.hh

```diff
@@ -1301,6 +1301,10 @@ public:
 
         future<> add_cf(table_id id, ks_cf_t kscf, lw_shared_ptr<column_family> cf);
         future<> remove_cf(table_id id, ks_cf_t kscf) noexcept;
+        column_family& get_table(table_id id) const;
+        const table_id& get_table_id(std::pair<std::string_view, std::string_view> kscf) const;
```

Comment:
can return table_id by value.
can accept kscf by const&
(should we also have a method that takes them as 2 separate params?)